### ISSUE TITLE
Web - Sweat and Toil - Fix Country Pages Accessibility Issues

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#97]
We need to fix the accessibility issues found on the country pages. An example page would be "Brazil" page.

https://www.dol.gov/agencies/ilab/resources/reports/child-labor/brazil

**Comment by carolynhuang** [comment]
@jfc3-dol, this reminder is fine, but there is nothing the app development team can do to address this. It would be more appropriate to log this wherever DOL webpage accessibility issues are stored.

**Comment by jfc3-dol** [comment]
All the website Sweat and Toil content need to be accessible too. 

Currently, the page listed above has at least 15 accessibility issues which I found quickly using the Web AIM WAVE toolbar without doing much work beside clicking the icon for the plug-in in Firefox, and there is also one for Chrome.

Someone needs to review all the links in the applications to make sure they point to the correct pages, and some are being redirected because of the URL change when moving to Drupal and the others are not found because the page has moved or been removed.

**Comment by clguidi** [comment]
John, we will let the web team know about this issue. However, since the app team cannot address this issue, please close it here.

**Comment by jfc3-dol** [comment]
@clguidi,

I need to leave it open, so I remember to check each time the application is updated because it's an accessibility issue I found while testing.

**Comment by clguidi** [comment]
Hi @jfc3-dol , we will be working to address this issue with the Web team. Could you please close it here?

**Comment by jfc3-dol** [comment]
@clguidi 

I only close the issues after I have tested them and made sure they were corrected.

**Comment by jfc3-dol** [comment]
@clguidi and @pbhatt17 

I'm going to close this issue and the other web-related one since this is for the Sweat and Toil iOS application.
